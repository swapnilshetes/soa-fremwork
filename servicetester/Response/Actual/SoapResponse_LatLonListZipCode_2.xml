<?xml version="1.0" encoding="ISO-8859-1"?>
<SOAP-ENV:Envelope SOAP-ENV:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/"
	xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/">
	<SOAP-ENV:Body>
		<SOAP-ENV:Fault>
			<faultcode xsi:type="xsd:string">SERVER</faultcode>
			<faultactor xsi:type="xsd:string"></faultactor>
			<faultstring xsi:type="xsd:string">Error with one or more zip codes: </faultstring>
			<detail xsi:type="xsd:string">
				&lt;error&gt;Error: Zip code &quot;2500&quot; is not a valid US zip
				code&lt;/error&gt;
			</detail>
		</SOAP-ENV:Fault>
	</SOAP-ENV:Body>
</SOAP-ENV:Envelope>