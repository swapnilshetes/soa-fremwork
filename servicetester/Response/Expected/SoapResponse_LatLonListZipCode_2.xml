<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
	xmlns:aws="http://webservices.amazon.com/AWSFault/2005-15-09">
	<soapenv:Body>
		<soapenv:Fault>
			<faultcode>aws:Client.MissingParameter</faultcode>
			<faultstring>The request must contain the parameter Signature.</faultstring>
			<detail>
				<aws:RequestId>2ed34319-3549-46ec-9606-9c1271d029a4</aws:RequestId>
			</detail>
		</soapenv:Fault>
	</soapenv:Body>
</soapenv:Envelope>